scipy/constants/_codata.py

```diff
@@ -1921,6 +2096,37 @@ def parse_constants_2018toXXXX(d: str) -> dict[str, tuple[float, str, float]]:
 _aliases['electric constant'] = 'vacuum electric permittivity'
 
 
+_extra_alias_keys = ['natural unit of velocity',
+                     'natural unit of action',
+                     'natural unit of action in eV s',
+                     'natural unit of mass',
+                     'natural unit of energy',
+                     'natural unit of energy in MeV',
+                     'natural unit of mom.um',
+                     'natural unit of mom.um in MeV/c',
+                     'natural unit of length',
+                     'natural unit of time']
+
+# finally, insert aliases for values
+for k, v in list(_aliases.items()):
+    if v in _current_constants or v in _extra_alias_keys:
+        physical_constants[k] = physical_constants[v]
+    else:
+        del _aliases[k]
+
+
+# The following derived quantities are no longer included,
+# but sync values for backward compatibility
+c = physical_constants['speed of light in vacuum'][0]
+mu0 = physical_constants['vacuum mag. permeability'][0]
+epsilon0 = physical_constants['vacuum electric permittivity'][0]
+physical_constants['characteristic impedance of vacuum'] = (
+    math.sqrt(mu0 / epsilon0), 'ohm',
+    math.sqrt(mu0 / epsilon0) * 0.5 * (
+        physical_constants['vacuum mag. permeability'][2] / mu0
+        + physical_constants['vacuum electric permittivity'][2] / epsilon0))
```

Comment:
The comment says:

> The following derived quantities are no longer included,

No longer included where? I understand that these values are accessible as `constants.c`, etc..., so I understand that they need to be synced with the the values in `physical_constants`; I just don't understand the first part of the comment.

Also, I see that `'characteristic impedance of vacuum'` used to be listed as "exact", and now it is not. If I understand correctly, the quantity has always been defined as `math.sqrt(mu0 / epsilon0)` (and it still is), and the underlying quantities have always been measured (and still are). Do I understand correctly that none of these definitions have changes; it's just that now `'characteristic impedance of vacuum'` is listed listed differently in the table because it is not really appropriate to list it as "exact" when there is uncertainty in the underlying quantities? And so we try to do better than the table by computing the value/uncertainty from the values/uncertainties of the underlying quantities? Is it the only quantity like this?

Reply:
```suggestion
```

Reply:
Actually it looks like all these lines can be deleted. (https://github.com/scipy/scipy/pull/16071/files#r1780501562)